pkgs/os-specific/linux/kernel/min-kernel-branch.txt

```diff
@@ -0,0 +1 @@
+4.19
```

Comment:
We're never going to support custom kernels that are older than the oldest supported stable kernel, so why not just read the minimum version from the kernel.org JSON file?

That way, dropping an old kernel just means removing it from the JSON, which as we've seen is what people expect to be able to do anyway.

Reply:
That's a good point, updated accordingly! :)